app/routes/planet_routes.py

```diff
@@ -1,30 +1,89 @@
-from flask import Blueprint, abort, make_response
-from app.models.planet import planets
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.planet import Planet
+from ..db import db
 
 # created blueprint
-planets_bp = Blueprint("planets_bp",__name__, url_prefix=("/planets"))
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
```

Comment:
We didn't explicitly look at invalid/missing data scenarios for creating a record, but as it came up in Slack, think about what would happen here, and what response we might want to reply with.